packages/admin/cms-admin/src/common/image/ImageCrop.tsx

```diff
@@ -36,6 +36,8 @@ export const ImageCrop = (props: ImageCropProps): React.ReactElement => {
                         crop={disabled ? {} : { ...value, unit: "%" }}
                         ruleOfThirds={true}
                         disabled={disabled}
+                        style={{ marginLeft: "40px" }}
```

Comment:
Why is this necessary? And shouldn't it be a padding on ImageContainer instead?

Reply:
It is used to align the image on the top left corner

Reply:
We removed the padding for the DialogContent [here](https://github.com/vivid-planet/comet/blob/main/packages/admin/cms-admin/src/blocks/image/EditImageDialog.tsx#L58-L62) to achieve the previous, now outdated design. Maybe it's a good idea to re-add it?